lib/pliny/metrics/backends/librato.rb

```diff
@@ -0,0 +1,50 @@
+begin
+  require "librato/metrics"
+  require "concurrent"
+rescue LoadError => error
+  puts <<-MSG
+! #{error.message}
+! Please add:
+!   gem "librato-metrics"
+!   gem "concurrent-ruby"
+! to your gemfile and run `bundle install` if you wish to use the Librato
+! metrics backend
+
+  MSG
+
+  raise
+end
```

Comment:
I mentioned this when we discussed this, but I believe this should be in a separate gem, for example `pliny-librato`